addons/account/security/account_security.xml

```diff
@@ -124,31 +124,31 @@
     <record id="account_move_comp_rule" model="ir.rule">
         <field name="name">Account Entry</field>
         <field name="model_id" ref="model_account_move"/>
-        <field name="domain_force">['|',('company_id','=',False),('company_id', 'in', company_ids)]</field>
+        <field name="domain_force">[('company_id', 'in', company_ids + [False])]</field>
     </record>
 
     <record id="account_move_line_comp_rule" model="ir.rule">
         <field name="name">Entry lines</field>
         <field name="model_id" ref="model_account_move_line"/>
-        <field name="domain_force">['|',('company_id','=',False),('company_id', 'in', company_ids)]</field>
+        <field name="domain_force">[('company_id', 'in', company_ids + [False])]</field>
     </record>
 
     <record id="journal_group_comp_rule" model="ir.rule">
         <field name="name">Journal Group multi-company</field>
         <field name="model_id" ref="model_account_journal_group"/>
-        <field name="domain_force">['|',('company_id','=',False),('company_id', 'in', company_ids)]</field>
+        <field name="domain_force">[('company_id', 'in', company_ids)]</field>
```

Comment:
Is it fine, that there is no company_ids + [False] here?
@KangOl

Reply:
> Is it fine, that there is no company_ids + [False] here?

@em230418 Yes, the field is required
https://github.com/odoo/odoo/blob/14.0/addons/account/models/account_journal.py#L20